public/javascripts/index.js

```diff
@@ -0,0 +1,19 @@
+var app = angular.module('release-master', ['ng']);
+
```

Comment:
Nope, just a staring point, will refactor later. This is just get it to work first